GrayBMP.cs

```diff
@@ -126,13 +126,28 @@ public void Dirty ()
       End ();
    }
 
+   /// <summary> Draws a line with given pixel thickness</summary>
+   public void DrawThickLine (int x0, int y0, int x1, int y1, int width, int color) {
+      Point2D startPt = new (x0, y0),
+              endPt = new (x1, y1);
+      double offset = width * 0.5;
+      for (int i = 1; i <= offset; i++) {
+         var vertices = Point2D.GetPolygonVertices (startPt, endPt, i);

```

Comment:
You are drawing a series of concentric polygons. Is this faster than filling a single polygon? Can you benchmark both and let me know the relative timings.

Reply:
Filling a single polygon is faster than drawing concentric polygons.
Average time for drawing concentric polygons : 225 milliseconds
Average time for filling a single polygon : 150 milliseconds